source/_components/device_tracker.bt_home_hub_5.markdown

```diff
@@ -21,10 +21,13 @@ To use a BT Home Hub 5 router in your installation, add the following to your `c
 device_tracker:
   - platform: bt_home_hub_5
     host: 192.168.1.254
+    password: ADMINPASSWORD
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.